### DISCUSSION TITLE
Is the Size-aware TinyLFU implementation part of this library?

### DISCUSSION THREAD

**Discussion by super-shayan** [ben-manes/caffeine#1416] [Q&A] [answered]
I am unable to locate the algorithm, is it available?

**Comment by ben-manes** [comment, marked as answer]
Yes, see @ohadeytan's [size research fork](https://github.com/ohadeytan/caffeine/tree/size_research). It would be nice to someday upstream that work.

**Reply by ben-manes** [reply]
Sorry, a different [branch](https://github.com/ohadeytan/caffeine/tree/TOS_submission) has a README detailing how to run the size aware policies.

**Reply by super-shayan** [reply]
perfect! thanks alot!!
